### ISSUE TITLE
vega issues on 2.x-maintenance branch

### ISSUE THREAD

**Issue by manthey** [girder/girder#2941]
Vega did a minor release on Feb 26 which included an npm package of vega-util which has ES6 code in its package.  This causes the web client tests for any plugin that uses vega to fail (and causes some downstream plugins' tests to fail if they use one of those plugins).  This includes, for instance, the jobs plugin.

This could be solved for each plugin that uses vega by adding a webpack helper file (much as was done in PR #2882).  This could be targeted to just vega, e.g., `webpack.helper.js` would be:
```
module.exports = function (config) {
    config.module.rules.push({
        resource: {
            test: /\.js$/,
            include: [/node_modules\/vega-.*/]
        },
        use: [{
            loader: 'babel-loader',
            options: {
                presets: ['env']
            }
        }]
    });
    return config;
};
```

Is there a better approach (other than to stop using vega since its releases are flaky)?  I can make a PR that adds these webpack helper files.

Note that the master branch does *not* fail in the web client tests, just the 2.x-maintenance branch.

**Comment by zachmullen** [comment]
@jeffbaumes @waxlamp how do you guys feel about vega in general? Should we consider alternatives?

**Comment by manthey** [comment]
I'll not that we use it for the jobs graph.

**Comment by manthey** [comment]
Resolved via #2942.
